.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] As part of the project requirements, please make sure to add `test.md` file to your project and place the file's name in the `.gitignore` file here so it doesn't get pushed to the GitHub repo.